evm.md

```diff
@@ -783,6 +793,30 @@ These are just used by the other operators for shuffling local execution state a
          </accounts>
 ```
 
+The following operations help with loading account information from an external running client.
+This minimizes the amount of information which must be stored in the configuration.
+
+-   `#loadAccount` queries for account data from the running client.
+-   `#lookupCode` loads the code of an account into the `<code>` cell.
+-   `#lookupStorage` loads the value of the specified storage key into the `<storage>` cell.
+
+```k
+    syntax InternalOp ::= "#loadAccount"   Int
+                        | "#lookupCode"    Int
+                        | "#lookupStorage" Int Int
+ // ----------------------------------------------
+```
```

Comment:
I guess we can move these definitions to `node.md` now.

Reply:
`#lookupCode` and `#lookupStorage` can definitely go. `#loadAccount` will depend on what becomes of the CREATE2 case.